chains/solana/contracts/programs/ccip-offramp/src/instructions/v1/execute.rs

```diff
@@ -587,6 +604,24 @@ mod execution_state {
     }
 }
 
+pub fn verify_uncursed_cpi<'info>(
+    rmn_remote: AccountInfo<'info>,
+    rmn_remote_config: AccountInfo<'info>,
+    rmn_remote_curses: AccountInfo<'info>,
+    chain_selector: u64,
+) -> Result<()> {
+    let cpi_accounts = rmn_remote::cpi::accounts::InspectCurses {
+        config: rmn_remote_config,
+        curses: rmn_remote_curses,
+    };
+    let cpi_context = CpiContext::new(rmn_remote, cpi_accounts);
+    rmn_remote::cpi::verify_not_cursed(
+        cpi_context,
+        CurseSubject::from_chain_selector(chain_selector),
+    )?;
+    Ok(())
+}
```

Comment:
This seems to be repeated with the helper in the `commit.rs` file of the offramp, could you please dedupe them?